<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistrictSearchWidget</class>
 <widget class="QWidget" name="DistrictSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>260</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lDistrict">
     <property name="text">
      <string>District:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DistrictComboBox" name="cbDistrict">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="tbDistrictAdd">
     <property name="toolTip">
      <string>Add to the list</string>
     </property>
     <property name="icon">
      <iconset theme="list-add"/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListView" name="lvDistricts">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="3">
    <widget class="Line" name="lTop">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="wButtons" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbDistrictRemove">
        <property name="toolTip">
         <string>Remove selection from the list</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbDistrictClear">
        <property name="toolTip">
         <string>Clear the list</string>
        </property>
        <property name="icon">
         <iconset theme="edit-clear-list"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>163</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DistrictComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/DistrictComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
